e2e/etcdctlv3_test.go

```diff
@@ -93,6 +109,9 @@ func ctlV3PrefixArgs(clus *etcdProcessCluster, dialTimeout time.Duration) []stri
 		endpoints = strings.Join(es, ",")
 	}
 	cmdArgs := []string{"../bin/etcdctlv3", "--endpoints", endpoints, "--dial-timeout", dialTimeout.String()}
+	if !quorum {
+		cmdArgs = append(cmdArgs, "--consistency", "s")
```

Comment:
prefix args probably shouldn't have a `quorum` argument since not all v3 commands have linearizable/serializable options-- `ctlV3Get` can append the consistency flag on its own